packages/govern-contract-utils/contracts/adaptative-erc165/AdaptativeERC165.sol

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.6.8;
+
+import "../erc165/ERC165.sol";
+
+contract AdaptativeERC165 is ERC165 {
```

Comment:
Did we test gas performance for this contract? I haven't, but we should make sure `suppportsInterface` spends less than 30k gas as per [the spec](https://eips.ethereum.org/EIPS/eip-165).

Reply:
good point. this really only does 1 SLOAD which is under 1k gas rn